libs/akita/src/lib/store.ts

```diff
@@ -299,6 +303,9 @@ export class Store<S = any> {
       this._initialState = initialState;
     }
     isDev() && assertStoreHasName(this.storeName, this.constructor.name);
+    this.ttlQueue.expired$.pipe(filter((ttl) => ttl.type === TTLType.Store)).subscribe(() => {
```

Comment:
Extract to a method and call if only if users enabled the cache feature, please.